src/main/kotlin/com/hibob/academy/employeeFeedback/resource/FeedbackResource.kt

```diff
@@ -34,15 +36,17 @@ class FeedbackResource(private val feedbackService: FeedbackService, private val
 
     @GET
     @Path("/allFeedback")
-    fun viewFeedback(@Context request: ContainerRequestContext): Response {
+    fun viewFeedback(@RequestBody filterBy: FeedbackFilterRequest, @Context request: ContainerRequestContext): Response {
         val employeeId = authenticatedUsersService.getLoggedInEmployeeId(request)
         val companyId = authenticatedUsersService.getLoggedInCompanyId(request)
         val role = authenticatedUsersService.getLoggedInRole(request)
         val validRoles = listOf(RoleType.ADMIN, RoleType.HR)
+        val anonymity = authenticatedUsersService.convertToAnonymityType(filterBy.anonymity)
 
         authenticatedUsersService.validateRole(role, validRoles)
 
-        val allFeedback = feedbackService.viewAllSubmittedFeedback(employeeId, companyId)
+        val filter = FeedbackFilterBy(filterBy.date, filterBy.department, anonymity)
```

Comment:
you can do this as a small fix later, but FeedbackFilterBy is also a bit confusing. I think I commented this before, these names sound like actions, but actually they just hold data.
The standard in bob is to call it a request object - so FeedbackFilterRequest, or something like this.
This makes it clear that it's only data